www/scripts/addversion.js

```diff
@@ -0,0 +1,129 @@
+const fs = require('fs');
+const path = require('path');
+const https = require('https');
+const url = require('url');
+const agent = new https.Agent({
+  keepAlive: true,
+});
+
+function get(fetchUrl) {
+  console.log('fetchUrl ', fetchUrl);
+  const { hostname } = url.parse(fetchUrl);
+  const options = {
+    agent: agent,
+    hostname: hostname,
+    path: fetchUrl,
+    headers: {
+      Accept: 'application/json',
+    },
+  };
+
+  return new Promise((accept, reject) => {
+    https.get(options, accept).on('error', reject);
+  });
+}
+
+function bufferStream(stream) {
+  return new Promise((accept, reject) => {
+    const chunks = [];
+    stream
+      .on('error', reject)
+      .on('data', (chunk) => chunks.push(chunk))
+      .on('end', () => accept(Buffer.concat(chunks)));
+  });
+}
+
+(async () => {
+  try {
+    const rootPath = path.resolve(__dirname, '..', '..');
+    const file = fs
+      .readFileSync(
+        path.join(rootPath, 'packages/crayons-core', 'package.json')
+      )
+      .toString();
+
+    const value = JSON.parse(file);
+
+    const key = value.version;
+
+    const versionsArr = JSON.parse(fs.readFileSync(process.env.VERSION_FILE));
```

Comment:
can you also add a default value if process.env.VERSION_FILE is not set

Reply:
added.